projects/controllable_dialogue/README.md

```diff
@@ -165,7 +165,7 @@ To talk to the "z=10 (boost)" version mentioned in the paper:
 
     python projects/controllable_dialogue/interactive.py \
     -mf models:controllable_dialogue/control_questionb11e10 \
-    -wd extrep_2gram:-3.5,extrep_nonstopword:-1e20,intrep_nonstopword:-1e20 \
```

Comment:
whoa what!

Reply:
what do you mean? I'm fixing a typo here. the amended version matches the config of what we mturked:

# Compared to inquisitive_model_ct_setting10, this removes extrep_2gram control
# (because it blocks questions), and adds beam reordering
# (i.e. given the top 10 candidates from beam search, choose the one which has lowest extrep_2gram).
# This should give much closer to 100% questions.
inquisitive_model_ct_setting10_better = {
    **inquisitive_model_ct_setting00,
    'set_controls': 'question:10',
    'beam_features_wts': 'extrep_nonstopword:-1e20,intrep_nonstopword:-1e20',
    'beam_reorder': 'best_extrep2gram_qn',
}